src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `books` (
+`id` BIGINT NOT NULL AUTO_INCREMENT,
+`title` VARCHAR(255),
+`price` INT,
+PRIMARY KEY (`id`)
+);
```

Comment:
always review your PR and add an empty line at the end of each file
![image](https://github.com/mate-academy/jv-jdbc-intro/assets/49767049/b5cf7470-6b4b-48f5-827e-da08a2af0b2b)